collections/front-end-javascript-frameworks/index.md

```diff
@@ -26,6 +26,7 @@ items:
  - preactjs/preact
  - ionic-team/stencil
  - withastro/astro
+ - vercel/next.js
```

Comment:
The collection changes would have been appended to the existing item list.

Reply:
thanks